Podcast/PlayerViewController.swift

```diff
@@ -321,13 +321,19 @@ class PlayerViewController: TabBarAccessoryViewController, PlayerDelegate, Playe
         episode.recommendedChange(completion: controlsView.setRecommendButtonToState)
     }
     
+    func didReceiveDownloadUpdateFor(episode: Episode) {
+
```

Comment:
can you download in player?

Reply:
Yes but the didReceive doesn't update the player's view at all.